### PR TITLE
Add piwik tracking

### PR DESCRIPTION
**Pull request by k-nut**
This PR lets us track usage so we can see if people are interested in this at all and which cities are the most active. We could also for example see if people are searching for cities that are not available and try to add those.

The tool we are using is [Piwik](https://piwik.org/) which is hosted with the Open Knowledge Foundation Germany so we do not have to fear that our users data is exposed to some kind of search giant.

I will share the credentials to piwik with anyone who is interested in private.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,6 +35,20 @@
...
         </div>
+        <!-- Piwik -->
+        <script type="text/javascript">
+          var _paq = _paq || [];
+          _paq.push(['trackPageView']);
+          _paq.push(['enableLinkTracking']);
+          (function() {
+            var u="//piwik.okfn.de/";
+            _paq.push(['setTrackerUrl', u+'piwik.php']);
+            _paq.push(['setSiteId', 18]);
```
Comment (johnjohndoe) [inline index.html]: Where is the magic number `18` coming from?

File: index.html [hunk 2]
```diff
@@ -35,6 +35,20 @@
...
+            var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
```
Comment (johnjohndoe) [inline index.html]: This code is hard to read because of these one letter variable names. Also I would put each statement in separate lines.

**Comment by johnjohndoe** [comment]
Please stick to the indentation of the file.
Would it make sense to move the script into its own .js file?

File: index.html (hunk 1 above)
Comment (k-nut) [inline index.html]: It is the 18th site that is tracked on that piwik instance as far as I can tell. So this is out of our control.

File: index.html (hunk 2 above)
Comment (k-nut) [inline index.html]: This is just copy pasted from the piwik setup page. I am much in favor of not changing this as this is the standard way to include piwik snippets.

**Comment by k-nut** [comment]
@johnjohndoe so change the indent to 4 spaces?

No, let us not move this to a separate file. The main code already is in a separate file that is loaded here but all of the included code is needed for setup. And that cannot be moved elsewhere.

I agree that including a blob of weirdly called javascript in your html pages is not pretty but unfortunately this is just how it is done with piwik (or google analytics for that matter).

File: index.html (hunk 1 above)
Comment (johnjohndoe) [inline index.html]: You could create a variable for it which has a pleasant name such as `var okfnPiwikId = 18;`

**Comment by johnjohndoe** [comment]
Yes, please use the same intent pattern as the rest of the file.

**Comment by k-nut** [comment]
@johnjohndoe did that :)

**Comment by dirkschumacher** [comment]
Just :shipit: I'd say :)

**Pull request merged**